Mathlib/CategoryTheory/Limits/Shapes/Opposites/Equalizers.lean

```diff
@@ -38,4 +39,343 @@ instance hasCoequalizers_opposite [HasEqualizers C] : HasCoequalizers Cᵒᵖ :=
     hasLimitsOfShape_of_equivalence walkingParallelPairOpEquiv
   hasColimitsOfShape_op_of_hasLimitsOfShape
 
+/-- The canonical isomorphism relating `parallelPair f.op g.op` and `(parallelPair f g).op` -/
+def parallelPairOpIso {X Y : C} (f g : X ⟶ Y) :
+    parallelPair f.op g.op ≅ walkingParallelPairOpEquiv.functor ⋙ (parallelPair f g).op :=
+  NatIso.ofComponents (fun
+    | .zero => .refl _
+    | .one => .refl _)
+    (by rintro (_ | _ | _) (_ | _ | _) f <;> cases f <;> cat_disch)
+
+@[simp]
+lemma parallelPairOpIso_hom_app_zero {X Y : C} (f g : X ⟶ Y) :
+  (parallelPairOpIso f g).hom.app WalkingParallelPair.zero = 𝟙 _ := rfl
+
+@[simp]
+lemma parallelPairOpIso_hom_app_one {X Y : C} (f g : X ⟶ Y) :
+  (parallelPairOpIso f g).hom.app WalkingParallelPair.one = 𝟙 _ := rfl
+
+@[simp]
+lemma parallelPairOpIso_inv_app_zero {X Y : C} (f g : X ⟶ Y) :
+  (parallelPairOpIso f g).inv.app WalkingParallelPair.zero = 𝟙 _ := rfl
+
+@[simp]
+lemma parallelPairOpIso_inv_app_one {X Y : C} (f g : X ⟶ Y) :
+  (parallelPairOpIso f g).inv.app WalkingParallelPair.one = 𝟙 _ := rfl
+
+/-- The canonical isomorphism relating `(parallelPair f g).op` and `parallelPair f.op g.op` -/
+def opParallelPairIso {X Y : C} (f g : X ⟶ Y) :
+    (parallelPair f g).op ≅ walkingParallelPairOpEquiv.inverse ⋙ parallelPair f.op g.op :=
+  calc
+    (parallelPair f g).op ≅ 𝟭 _ ⋙ (parallelPair f g).op := .refl _
+    _ ≅ (walkingParallelPairOpEquiv.inverse ⋙ walkingParallelPairOpEquiv.functor) ⋙ _ :=
+      isoWhiskerRight walkingParallelPairOpEquiv.symm.unitIso _
+    _ ≅ walkingParallelPairOpEquiv.inverse ⋙ walkingParallelPairOpEquiv.functor ⋙ _ :=
+      Functor.associator _ _ _
+    _ ≅ walkingParallelPairOpEquiv.inverse ⋙ parallelPair f.op g.op :=
+      isoWhiskerLeft _ (parallelPairOpIso f g).symm
+
+@[simp]
+lemma opParallelPairIso_hom_app_zero {X Y : C} (f g : X ⟶ Y) :
+  (opParallelPairIso f g).hom.app (op WalkingParallelPair.zero) = 𝟙 _ := by simp [opParallelPairIso]
+
+@[simp]
+lemma opParallelPairIso_hom_app_one {X Y : C} (f g : X ⟶ Y) :
+  (opParallelPairIso f g).hom.app (op WalkingParallelPair.one) = 𝟙 _ := by simp [opParallelPairIso]
+
+@[simp]
+lemma opParallelPairIso_inv_app_zero {X Y : C} (f g : X ⟶ Y) :
+  (opParallelPairIso f g).inv.app (op WalkingParallelPair.zero) = 𝟙 _ := by simp [opParallelPairIso]
+
+@[simp]
+lemma opParallelPairIso_inv_app_one {X Y : C} (f g : X ⟶ Y) :
+  (opParallelPairIso f g).inv.app (op WalkingParallelPair.one) = 𝟙 _ := by simp [opParallelPairIso]
+
+namespace Cofork
+
+/-- The obvious map `Cofork f g → Fork f.unop g.unop` -/
+def unop {X Y : Cᵒᵖ} {f g : X ⟶ Y} (c : Cofork f g) : Fork f.unop g.unop :=
+   Cocone.unop ((Cocones.precompose (opParallelPairIso f.unop g.unop).hom).obj
+      (Cocone.whisker walkingParallelPairOpEquiv.inverse c))
+
+lemma unop_π_app_one {X Y : Cᵒᵖ} {f g : X ⟶ Y} (c : Cofork f g) :
+    c.unop.π.app .one = (Quiver.Hom.unop (c.ι.app .zero)) := by
+  simp [unop, walkingParallelPairOpEquiv]
```

Comment:
```suggestion
  simp [unop]
```

With the extra lemmas added about this equivalence, it’s no longer necessary to add it explicitly to the simp set, this applies in a few places below.